lib/apis/moray.js

```diff
@@ -1598,8 +1598,7 @@ Moray.prototype.getVmMigrations = function (params, cb) {
             return;
         }
         if (param === 'state' && params[param] === 'active') {
-            // Active is considering 'running', 'paused' or 'failed'.
-            filter += '(|(state=running)(state=paused)(state=failed))';
+            // Active is deprecated - ignore this param and return all states.
         } else if (param === 'source_server_uuid') {
```

Comment:
I understand that the idea is to make a note for the future that we're ignoring what we previously didn't. I'd rather comment out the code instead of removing the line, and comment out the first branch of the conditional altogether, instead of leaving an empty conditional branch, which will be evaluated always.

Reply:
Note this same empty conditional branch is what's making the build fail:

```
22:00:10  /root/data/jenkins/workspace/joyent-org_sdc-vmapi_TRITON-2110/lib/apis/moray.js(1600): warning: empty statement or extra semicolon
```

Reply:
It's not really a note for future - it actually still needs to be handled in the code. If I just comment out the code then 'state=active' ends up in the Moray query, which will result in no results.

The code is valid and deliberately empty (stoopid jsl).

Reply:
Updated to work around the jsstyle nit.